docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc

```diff
@@ -650,7 +664,83 @@ public class TokenResource {
 }
 ```
 
-In your own endpoint you can access individual `UserInfo` properties as required.
+If you support more than one social provider with the help of link:security-openid-connect-multitenancy[OpenId Connect Multi-Tenancy], for example, `Google` which is an OpenId Connect Provider returning `IdToken` and `GitHub` which is an `OAuth2` provider returning no `IdToken` and only allowing to access `UserInfo` then you can have your endpoint working with only the injected `SecurityIdentity` for both `Google` and `GitHub` flows. A simple augmentation of `SecurityIdentity` will be required where a principal created with the internally generated `IdToken` will be replaced with the `UserInfo` based principal when the GiHub flow is active:
+
+```java
```

Comment:
Could you use proper Asciidoc code blocks in your doc? I fixed them in the latest pull requests but it would be nice if you could switch to them once and for all.

It should be:
```
[source,java]
----
your code
----
```

Reply:
@gsmet Is what is used in this PR not correct `Asciidoc` ? In fact I used to always do `source.,...` but then I saw somewhere the shortcuts and started using it - what is the problem with it ?